TADashboard/src/main/java/page_objects/GeneralSettingsPage.java

```diff
@@ -0,0 +1,25 @@
+package page_objects;
+
+import com.logigear.control.common.imp.Button;
+import com.logigear.driver.DriverUtils;
+
+public class GeneralSettingsPage {
+    private Button btnNext = new Button("css=input[value='Next']");
+    private Button btnFinish = new Button("css=input[value='Finish']");
+
+    public void clickBtnNext() {
+        btnNext.click();
+    }
+
+    public void clickBtnFinish() {
+        btnFinish.click();
+    }
+
+    public String isAlertVisible() {
```

Comment:
it should be getAlertText() instead
only use `is...()` or `does....()` method if it returns boolean

Reply:
I fixed it. Please check again.